PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackCuts.h

```diff
@@ -207,6 +207,13 @@ class AliFemtoDreamTrackCuts {
     fCutPID = kTRUE;
   }
   ;
+  void SetCutITSPID(float pITSThresh = 0.0, double sigValITSmin = -3., double sigValITSmax = 3., bool cutit = false) {
```

Comment:
why would you call that function and then via the cutit bool force not to use the cuts? 
remove the last bool, and once you call the function automatically set fdoITSnSigmaCut = true;